MIGRATION.md

```diff
@@ -1,5 +1,28 @@
 # Migration guides
 
+## 3.5.12 to 3.5.13
+- Custom events have been renamed to conversions.
+      - `Radar.sendEvent(customType:metadata:completionHandler:)` is now `Radar.logConversion(name:metadata:completionHandler)`.
+      - `Radar.logConversion(name:revenue:metadata:callback:)` has been added.
+      - `Radar.sendEvent(customType:metadata:location:callback:)` has been removed.
+      - `RadarSendEventCompletionHandler(status, location, events, user)` is now `RadarLogConversionCompletionHandler(status, event)`.
+            - `location` and `user` are no longer available, and only the conversion event is returned as `event` instead of a coalesced list of events.
+      - On `RadarEvent`, `customType` is now `conversionName`, and `RadarEventType.custom` is now `RadarEventType.conversion`.
+
+```swift
+// 3.5.13
```

Comment:
If you scroll back through some of the other entries in MIGRATION.MD you can see the pattern we want to use for the code snippets.  Basically an example with the new version of the SDK and then underneath that an example on the old version.